eng/common/build.sh

```diff
@@ -137,13 +137,16 @@ while [[ $# > 0 ]]; do
       node_reuse=$2
       shift
       ;;
-    /p:*)
+    -p:*|/p:*)
       properties="$properties $1"
       ;;
-    /m:*)
+    -m:*|/m:*)
       properties="$properties $1"
       ;;
-    /bl:*)
+    -bl:*|/bl:*)
+      properties="$properties $1"
+      ;;
+    -dl:*|/dl:*)
```

Comment:
This change will need to be made in Arcade as well so we don't get out of sync. I'll put that together.

Reply:
https://github.com/dotnet/arcade/pull/2191

Reply:
It seems like you already know and are just trying to stay unblocked, but I'll say it for posterity: Don't edit this folder by hand: https://github.com/aspnet/Extensions/blob/master/eng/common/README.md. Instead make a PR to arcade then get the new version when it's merged. Unblocking development temporarily as @anurse has done here by editing local files is fine if you have a PR out and are waiting for it to be merged, but DO NOT merge it that way.